cli/config/config.go

```diff
@@ -94,9 +95,8 @@ func GetComposeFile(isProfile bool, composeName string) (string, error) {
 		"composeFilePath": composeFilePath,
 		"error":           err,
 		"type":            serviceType,
-	}).Trace("Compose file not found at workdir. Extracting from binary resources")
+	}).Tracef("Compose file not found at %s. Please make sure the file exists at the %s location", filepath.Dir(composeFilePath), composeFilePath)
```

Comment:
Because the log fields already display the path:
```suggestion
	}).Trace("Compose file not found. Please make sure the file exists at the location")
```